thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/route.js

```diff
@@ -59,10 +59,14 @@ const severityMap = {
  * If no filter data is set for sensitivity, use this
  */
 const sensitivityDefaults = {
+  defaultMttdVal: '5',
   selectedSeverityOption: 'Percentage of Change',
   selectedTunePattern: 'Up and Down',
   defaultPercentChange: '0.3',
-  defaultMttdChange: '5'
+  mttdGranularityMinimums: {
+    days: 24,
```

Comment:
Hard to understand here, why we set the days to be "24"?

Reply:
Oh, I see what you're saying here. I need to base these limits on `windowSize` and `windowUnit`. Ok - will modify.

Reply:
The granularity minimums are max number of hours (MTTD)